### DISCUSSION TITLE
Adding a

### DISCUSSION THREAD

**Discussion by 0xjams** [firefly-iii/firefly-iii#9039] [General]
My bank has this structure:

```
No,Fecha,Ref.,Lugar,Detalle,Secuencial,(+/-),Valor,Saldo Disponible,Saldo Contable,Descripci�n
1,08/07/2024,xxx,0-xxx,xxx,'xxx,(-),0.25,"xxx.xx","xxx.xx",Description
```
The `Amount` column is the one called `Valor`. However, the previous column (+/-) indicates if it's debit or credit. I noticed that there's a mapping that helps with this: `generic-debit-credit`. I tried to use that mapper, but all of the values were sent as positive.

Could I solve this by sending a pull request that adds `(-)` to this array in `BankDebitCredit.php`?

```
        $negative = [
            'D', // Old style Rabobank (NL). Short for "Debit"
            'A', // New style Rabobank (NL). Short for "Af"
            'DR', // https://old.reddit.com/r/FireflyIII/comments/bn2edf/generic_debitcredit_indicator/
            'Af', // ING (NL).
            'Debet', // Triodos (NL)
            'Debit', // ING (EN), thx Quibus!
            'debit', // Mint
            'S', // Volksbank (DE), Short for "Soll"
            'DBIT', // https://subsembly.com/index.html (Banking4 App)
            'Charge', // not sure which bank but it's insane.
        ];
```

**Comment by JC5** [comment]
Yes, that should solve it. Reminds me to make that configurable instead :)

**Comment by 0xjams** [comment]
Hi @JC5 

I just sent a [pull request](https://github.com/firefly-iii/data-importer/pull/638)
